lib/schema/core_facts.pot

```diff
@@ -0,0 +1,1354 @@
+# CORE FACTS SCHEMA
+# Copyright (C) 2016 Puppet Labs, LLC
+# This file is distributed under the same license as the Sinatra i18n demo package.
+# FIRST AUTHOR <info@puppet.com>, 2016.
+#
+#, fuzzy
+# msgid ""
+# msgstr ""
+"Project-Id-Version: Sinatra i18n demo init-11-ga552a06\n"
+"\n"
+"Report-Msgid-Bugs-To: docs@puppetlabs.com\n"
+"POT-Creation-Date: 2016-08-26 09:00-0000\n"
+"PO-Revision-Date: 2016-08-26 11:50:55 -0700\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"
+  
+
+# aio_agent_version description
+msgid "Return the version of the puppet-agent package that installed facter."
+msgstr ""
+
+# aio_agent_version resolution
+msgid "All platforms: use the compile-time enabled version definition."
+msgstr ""
+
+# architecture description
+msgid "Return the operating system's hardware architecture."
+msgstr ""
+
+# architecture resolution
+msgid "POSIX platforms: use the `uname` function to retrieve the OS hardware architecture.Windows: use the `GetNativeSystemInfo` function to retrieve the OS hardware architecture."
+msgstr ""
+
+# architecture caveats
+msgid "Linux: Debian, Gentoo, kFreeBSD, and Ubuntu use "amd64" for "x86_64" and Gentoo uses "x86" for "i386"."
+msgstr ""
+
+# augeas description
+msgid "Return information about augeas."
+msgstr ""
+
+# augeas resolution
+msgid "All platforms: query augparse for augeas metadata."
+msgstr ""
+
+# augeasversion description
+msgid "Return the version of augeas."
+msgstr ""
+
+# augeasversion resolution
+msgid "All platforms: query augparse for the augeas version."
+msgstr ""
+
+# blockdevices description
+msgid "Return a comma-separated list of block devices."
+msgstr ""
+
+# blockdevices resolution
+msgid "Linux: parse the contents of `/sys/block/<device>/`.Solaris: use the `kstat` function to query disk information."
```

Comment:
I mean right now, all the entries under the `resolution` key are concatenated.

If the script to generate a webpage from the schema does lookup on each entry, i.e. something like
```
schema.each do |fact|
  fact['resolution'].each do |resolution|
    puts resolution
  end
end
```
then lookup for the whole string will fail. So this is an area where your script and the docs script to generate webpages need to align.